src/routing/tests/mod.rs

```diff
@@ -164,7 +159,7 @@ async fn receive_mismatching_get_section_request_as_adult() -> Result<()> {
     assert_matches!(
         message,
         SectionInfoMsg::GetSectionResponse(GetSectionResponse::Redirect(addrs)) => {
-            assert_eq!(addrs, elders_addrs)
+            assert_eq!(addrs, section_auth)
```

Comment:
Can rename addrs to received_auth to reduce confusion.

Reply:
ah yeah, sure I'll do that!